locales/it-IT/protondb-content.json

```diff
@@ -188,48 +188,48 @@
       "inWishlist": "In Wishlist",
       "mostBorked": "I Report Più Rotti!",
       "mostPopular": "Più popolari",
-      "percentPositive": "Well Reviewed on Store",
-      "playerCount": "Most Active Players",
-      "popular": "Most Steam Followers",
-      "rank": "Most Popular",
+      "percentPositive": "Recensito positivamente su Store",
+      "playerCount": "Con giocatori piú attivi",
+      "popular": "Con piú Steam Follower",
+      "rank": "Piú Popolare",
       "recentlyImproved": "Migliorati di recente",
-      "recentlyReleased": "Recently Released",
-      "releaseDate": "By Release Date",
-      "userScore": "Well Reviewed",
+      "recentlyReleased": "Recentemente rilasciati",
+      "releaseDate": "Per data di rilascio",
+      "userScore": "Recensito bene",
       "wellRatedHere": "Ben valutati qui"
     },
     "layout": "Layout",
     "loadError": "Errore nel caricamento!",
     "notLoggedIn": "Impossibile caricare la libreria senza uno SteamID. Visita il tuo [Profilo]({{link}}) per aggiungerlo.",
     "poweredBy": "Powered by",
     "sorts": {
-      "dateAdded": "Recently Added",
+      "dateAdded": "Recentemente Aggiunti",
       "fixWanted": "Fix necessario",
-      "heavilyReported": "Recent Report Tally",
+      "heavilyReported": "Conteggio dei report recenti",
       "hoursPlayed": "Ore giocate",
       "mostBorked": "Più rotti",
-      "percentPositive": "Percent Positive",
+      "percentPositive": "Percentuali Positive",
       "playerCount": "Numero di giocatori",
-      "popular": "Popular",
-      "rank": "Most Popular",
+      "popular": "Popolare",
+      "rank": "Piú Populare",
       "recentlyImproved": "Migliorato di recente",
-      "recentlyReleased": "Recently Released",
-      "releaseDate": "Release Date",
+      "recentlyReleased": "Recentemente Rilasciati",
+      "releaseDate": "Data di Rilascio",
       "userScore": "Punteggio degli utenti",
       "wilsonRating": "Valutazione di ProtonDB"
     },
     "tagData": "Tag data",
     "toggles": {
       "antiCheat": "Anti-Cheat",
       "deckVerified": "Deck Verified",
-      "excludeNative": "Exclude Native",
+      "excludeNative": "Escludi Nativo",
       "includeNative": "Includi Nativo",
       "lackReports": "Report Mancanti",
       "restrictToLibrary": "Libreria",
       "untested": "Report Mancanti",
       "userTags": "Tag degli utenti",
-      "whitelisted": "Whitelisted",
-      "wishlisted": "Wishlisted"
+      "whitelisted": "Escluso",
```

Comment:
I am not sure about that term what means in protondb

Reply:
i think you figured this out already, but this corresponds to 'Deck Verified' in the english localization. unfortunately now we have another kind of whitelist, the chromebook ready kind. but i'd rather not change the locale key over it, for now.

Reply:
yeah I updated with a commit